content/docs/integrating-with-other-libraries.md

```diff
@@ -39,21 +39,21 @@ class SomePlugin extends React.Component {
 }
 ```
 
-Note that we defined both `componentDidMount` and `componentWillUnmount` [lifecycle methods](/docs/react-component.html#the-component-lifecycle). Many jQuery plugins attach event listeners to the DOM so it's important to detach them in `componentWillUnmount`. If the plugin does not provide a method for cleanup, you will probably have to provide your own, remembering to remove any event listeners the plugin registered to prevent memory leaks.
+Not edin ki, biz hem `componentDidMount` hem de `componentWillUnmount` [lifecycle methods](/docs/react-component.html#the-component-lifecycle)'i tanımladık. Birçok jQuery eklentisi, olay dinleyicilerini DOM'a ekler, bu nedenle onları `componentWillUnmount`'tan ayırmak önemlidir. Eğer eklenti düzeltme için bir yöntem sağlamıyorsa, muhtemelen hafıza sızıntılarını önlemek için eklentinin kaydettiği herhangi bir olay dinleyicisini kaldırmayı hatırlayarak kendi önleminizi almak zorunda olacaksınız. 
 
 ### Integrating with jQuery Chosen Plugin {#integrating-with-jquery-chosen-plugin}
 
-For a more concrete example of these concepts, let's write a minimal wrapper for the plugin [Chosen](https://harvesthq.github.io/chosen/), which augments `<select>` inputs.
+Bu kavramların daha kesin bir örneği için, `<select>` girdileri arttırılmıs [Chosen](https://harvesthq.github.io/chosen/) eklentisi için küçük bir sarmalayıcı yazalım.
 
->**Note:**
+>**Not:**
 >
->Just because it's possible, doesn't mean that it's the best approach for React apps. We encourage you to use React components when you can. React components are easier to reuse in React applications, and often provide more control over their behavior and appearance.
+>Bunun mümkün olması, React uygulamaları için en iyi yaklaşım olduğu anlamına gelmez. Biz, yapabildiğin zaman, seni React bileşenlerini kullanman için yüreklendiriyoruz. React uygulamalarında, react bileşenlerini yeniden kullanmak daha kolaydır, ve sıklıkla davranışları ve görünümleri üzerinde daha fazla kontrol sağlar.
 
-First, let's look at what Chosen does to the DOM.
+Öncelikle, `Chosen`'ın DOM'a ne yaptığına bakalım.
 
-If you call it on a `<select>` DOM node, it reads the attributes off of the original DOM node, hides it with an inline style, and then appends a separate DOM node with its own visual representation right after the `<select>`. Then it fires jQuery events to notify us about the changes.
+Eğer bunu bir `<select>` DOM ​​düğümünde çağırırsanız, orijinal DOM düğümünün özniteliklerini okur, bir satır içi stil ile onu saklar, ve hemen `<select>`'ten sonra, kendi görsel temsili ile ayrı bir DOM düğümü ekler. Ardından, değişiklikleri bize bildirmek için jQuery olaylarını tetikler.
 
-Let's say that this is the API we're striving for with our `<Chosen>` wrapper React component:
+Diyelim ki, bu `<Chosen>` sarmalayıcı React bileşenimizle uğraştığımız API'dır:
```

Comment:
"sarmalayıcı" should be "sarmalayıcısı".

Reply:
Agreed